src/components/PlantingSites/PlantingSiteCreate.tsx

```diff
@@ -129,7 +129,7 @@ export default function CreatePlantingSite(props: CreatePlantingSiteProps): JSX.
   };
 
   return (
-    <TfMain>
+    <TfMain moreScreen={true}>
```

Comment:
I think I missed in visual changes what this moreScreen prop does?

Reply:
Every page with TfMain has 200px bottom padding which is a lot. This option reduces bottom padding to 80px so we can show a taller map

Reply:
Thanks for the explanation!